src/main/scala/EnsimePlugin.scala

```diff
@@ -553,14 +575,24 @@ case class EnsimeModule(
   runtimeJars: Set[File],
   testJars: Set[File],
   sourceJars: Set[File],
-  docJars: Set[File]
+  docJars: Set[File],
+  configurations: List[EnsimeConfiguration]
```

Comment:
But, the idea is the `EnsimeConfiguration`s are per module, their scope is within each sub project